nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -312,13 +330,13 @@ in
         gnome-flashback
       ] ++ (map
         (wm: gnome-flashback.mkSystemdTargetForWm {
-          inherit (wm) wmName;
+          inherit (wm) wmLabel wmCommand enableGnomePanel;
         }) cfg.flashback.customSessions);
 
-        # gnome-panel needs these for menu applet
-        environment.sessionVariables.XDG_DATA_DIRS = [ "${pkgs.gnome.gnome-flashback}/share" ];
-        # TODO: switch to sessionVariables (resolve conflict)
-        environment.variables.XDG_CONFIG_DIRS = [ "${pkgs.gnome.gnome-flashback}/etc/xdg" ];
+      # gnome-panel needs these for menu applet
+      environment.sessionVariables.XDG_DATA_DIRS = [ "${pkgs.gnome3.gnome-flashback}/share" ];
```

Comment:
This needs to be changed back to `gnome.

Reply:
Done.